bench/performance/benchmarks/update_bench.rb

```diff
@@ -41,15 +43,21 @@ class LogidzeUser
   x.config(time: BM_TIME, warmup: BM_WARMUP)
 
   x.report("Plain UPDATE #2") do
-    User.random.update!(params2)
+    user = User.random
+    user.update!(params2.except(*JSON_COLUMNS))
+    user.update!(params2.slice(*JSON_COLUMNS))
   end
 
   x.report("PT UPDATE #2") do
-    PaperTrailUser.random.update!(params2)
+    user = PaperTrailUser.random
+    user.update!(params2.except(*JSON_COLUMNS))
+    user.update!(params2.slice(*JSON_COLUMNS))
   end
 
   x.report("Logidze UPDATE #2") do
-    LogidzeUser.random.update!(params2)
+    user = LogidzeUser.random
+    user.update!(params2.except(*JSON_COLUMNS))
+    user.update!(params2.slice(*JSON_COLUMNS))
```

Comment:
As for why this change is needed: I didn't dig deep, but the error doesn't appear when we update multiple columns at once (the JSON and JSONB columns are saved as strings in `log_data` column, but apart from that everythin works as expected). However, if we update only JSON and JSONB columns, the error can be reproduced.